src/methods.js

```diff
@@ -1,35 +1,20 @@
 /* eslint-disable no-inline-comments */
-
-/**
- * Module dependencies.
- */
-
 import { map, set } from 'lodash';
 
 /**
  * Export available rpc methods.
  */
 
-
  /**
  * @Dev
- * to update the methods:
- * new : >= new_version
- * changed: >= new_version
- * deprecate < new_version
+ * follow the versioning rule below 
+ * to update changes:
```

Comment:
Thanks for adding this.    Also what's the deal with some of the exports below having >== vs. >=?    Any idea?

Reply:
I extracted the >= format based on the previous commits done on this file.  The `*==`  was added by Jag. 

@sidhujag any specific reason for using your format vs current format?